packages/twenty-server/src/engine/core-modules/quick-actions/quick-actions.service.ts

```diff
@@ -159,7 +161,7 @@ export class QuickActionsService {
     }
 
     const enrichedData = await this.intelligenceService.enrichCompany(
-      company.domainName,
+      getCompanyNameFromDomainName(getCompanyDomainName(company.domainName)),
```

Comment:
It used to work with domainName only, why do we want the name of the company now?

Reply:
its a mistake, thanks for spotting this. (i think this service is not in use anymore by the way)